packages/opentelemetry-plugin-grpc-js/src/grpcJs.ts

```diff
@@ -47,15 +47,6 @@ export class GrpcJsPlugin extends BasePlugin<typeof grpcJs> {
     return this._tracer;
   }
 
-  /**
-   * @internal
-   * Public reference to the protected BasePlugin `_logger` instance to be used by this
-   * plugin's external helper functions
-   */
-  get logger(): Logger {
```

Comment:
Turns out this was the only getter and after refactoring to only use api.diag, this appears to be a safe removal.